src/PlatformBenchmarks/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Net;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace PlatformBenchmarks
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            new WebHostBuilder()
+                .UseKestrel(options =>
+                {
+                    options.Listen(IPAddress.Loopback, 8080, builder =>
```

Comment:
It needs to use the address passed in `--server.urls` so I'd suggest to use `UseConfiguration(config)` and get the args and ENV added. I will do a PR to this PR to test it works.

Reply:
Lame 😄